cli/inspect.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// PsCommand is used to implement 'ps' command.
+type InspectCommand struct {
+	baseCommand
+}
+
+// Init initializes InspectCommand command.
+func (p *InspectCommand) Init(c *Cli) {
+	p.cli = c
+	p.cmd = &cobra.Command{
+		Use:   "inspect [container]",
+		Short: "get the detailed information of containers",
```

Comment:
I do not think current the code supports multiple container inspect. Maybe we should change to `Get detailed information of specified container`. And Args should be `Args:  cobra.ExactArgs(1),`.

In addition, we should add a RunE function for p.cmd, like others: https://github.com/alibaba/pouch/blob/master/cli/create.go#L23-L25. We need to keep consistent since there are one former pull request to make this.

Reply:
Yeah, I got it.